conductor-client/src/main/kotlin/com/openlattice/collaborations/PostgresCollaborationDatabaseService.kt

```diff
@@ -152,7 +152,6 @@ class PostgresCollaborationDatabaseService(
     override fun initializeTableProjection(collaborationId: UUID, organizationId: UUID, tableId: UUID) {
         val collaborationHds = externalDbConnMan.connectToOrg(collaborationId)
         val table = externalTables.getValue(tableId)
-        val columns = externalColumns.values(Predicates.equal(TABLE_ID_INDEX, tableId)).sortedBy { it.ordinalPosition }.toSet()
```

Comment:
This is because because permissions no longer exist for roles based on columns, yes?

Reply:
Yes, and also this was used only for `initializeProjectedTableViewPermissions`, which is originally in https://github.com/openlattice/openlattice/blob/c570a08a9d9ce7fb5aaed89b8c23ba8d73e8ad27/conductor-client/src/main/kotlin/com/openlattice/collaborations/PostgresCollaborationDatabaseService.kt#L183-L187 but is now removed.